mappings/1.8#1.8.1-pre1.tinydiff

```diff
@@ -214,14 +214,14 @@ c	net/minecraft/C_24qhtkqyz$1	1
 	m	(Ljava/lang/Object;)Z	apply	apply	
 		p	1		p_1	
 c	net/minecraft/C_25obpltwe
-	f	D	f_84dysutwo		f_84dysutwo
-	f	[D	f_41tptzxcz		f_41tptzxcz
-	f	[D	f_42sxgxbfu		f_42sxgxbfu
-	m	(DD)D	m_43fwsbzrl		m_43fwsbzrl
-		p	0			p_0
-		p	2			p_2
-	m	(D)D	m_97siwddxn		m_97siwddxn
-		p	0			p_0
+	f	D	f_84dysutwo		MAGIC_ATAN2_CONSTANT
+	f	[D	f_41tptzxcz		ASIN_TABLE
+	f	[D	f_42sxgxbfu		Y_TO_X_TABLE
```

Comment:
This looks like a cosine table to me, and a look at modern Yarn and Mojmaps suggests this is indeed the case.

Reply:
Well, it's a table that involves cosines. If you look at the graph of the function, it's pretty clear that it converts an x-coordinate on the unit circle to the corresponding positive y-coordinate.
![image](https://user-images.githubusercontent.com/116048347/204939455-da40d609-15b6-48e6-b3ef-46b1fa3a1469.png)